modules/priceFloors.js

```diff
@@ -55,7 +55,7 @@ export let _floorDataForAuction = {};
  * @summary Simple function to round up to a certain decimal degree
  */
 function roundUp(number, precision) {
-  return Math.ceil(parseFloat(number) * Math.pow(10, precision)) / Math.pow(10, precision);
+  return Math.ceil((parseFloat(number) * Math.pow(10, precision)).toFixed(1)) / Math.pow(10, precision);
```

Comment:
Can you explain how this solves the rounding issue?

Reply:
Nevermind, took a look and it works. Good job!

Reply:
The rounding issue was due to the JS evaluation of the number after multiplying the two values. So 0.6271 multiplied by 10000 would be set as 6271.0000000000000001. Running the ceiling on this will cause it to round up to 6272. By fixing to only one decimal allowed it to round this only to the first decimal place prior to running the ceiling, thus avoiding rounding up unnecessarily.

Reply:
why round instead of down? is this a convention of sorts?

Reply:
@patmmccann 

Say the actual floor is:

0.611

if we round to

0.61

and send that to the bidder, and they bid at 0.61, they will be floored.

So rounding up, makes sure we never run into this scenario.

Reply:
interesting, in the case of not enforcing, seems like it should be flipped. However, probably quite the edge case, fyi @punkiller